mlflow/pytorch/__init__.py

```diff
@@ -1084,13 +1084,16 @@ def print_auto_logged_info(r):
     import atexit
 
     try:
-        import pytorch_lightning as pl
+        import lightning as L
     except ImportError:
-        pass
-    else:
+        import pytorch_lightning as pl
         from mlflow.pytorch._lightning_autolog import patched_fit
 
         safe_patch(FLAVOR_NAME, pl.Trainer, "fit", patched_fit, manage_run=True)
```

Comment:
Can we do the following?

```python
    try:
        import pytorch_lightning as pl
    ...

    try:
        import lightning as L
    ...
```

Reply:
the `lightning` package internally installs `pytorch_lightning`. The import `import pytorch_lightning as pl` still works in 2.0. if we set the code this way, `safe_patch` would get executed twice.